frontend/packages/kubevirt-plugin/integration-tests/tests/cdi-upload.scenario.ts

```diff
@@ -0,0 +1,55 @@
+import * as path from 'path';
+import { browser, ExpectedConditions as until } from 'protractor';
+import { click } from '@console/shared/src/test-utils/utils';
+import { clickNavLink } from '@console/internal-integration-tests/views/sidenav.view';
+import {
+  createItemButton,
+  isLoaded,
+  saveChangesBtn,
+} from '@console/internal-integration-tests/views/crud.view';
+import { VM_CREATE_AND_EDIT_TIMEOUT_SECS } from './utils/constants/common';
+import * as cdiUploadView from '../views/cdiUploadView';
+import { inputPVCName, inputPVCSize } from '@console/internal-integration-tests/views/storage.view';
+import {
+  selectItemFromDropdown,
+  storageclassDropdown,
+} from '@console/ceph-storage-plugin/integration-tests/views/pvc.view';
+
+describe('KubeVirt CDI Upload', () => {
+  const pvc = { name: 'upload-pvc', size: 1, unit: 'Gi', storageClassName: 'rook-ceph' };
```

Comment:
the unit is not used. Anyway we could be using a smaller size, since 1Gi is a bit of overkill for a small text file

Reply:
need at least 100MB for the dv to initiate succesfully